### PR TITLE
Supports Public Joining of Public Initiatives

### PR DESCRIPTION
**Pull request by gbdubs**
Note the Authz for this is in #122

### CODE REVIEW

File: frontend/pages/initiative/[id]/relationships.vue [hunk 1]
```diff
@@ -28,15 +29,19 @@ const canManage = computed(() => {
...
-const makeManager = (userId: string) => {
-  alert('make manager not yet implemented')
-}
-const removeFromInitiative = (userId: string) => {
-  alert('remove from initiative not yet implemented')
-}
-const removeManager = (userId: string) => {
-  alert('remove manager not yet implemented')
-}
+const changeMembership = (userId: string, member: boolean | undefined, manager: boolean | undefined) =>
```
Comment (bcspragu) [inline frontend/pages/initiative/[id]/relationships.vue:32]: Doesn't look like `manager` is used here?

File: frontend/pages/initiative/[id]/index.vue [hunk 2]
```diff
@@ -26,6 +30,26 @@ const status = computed(() => {
     return 'Closed'
   }
 })
+
+const isAMember = computed(() => {
+  const mm = maybeMe.value
+  if (!mm) return false
+  return relationships.value.some((r) => r.userId === mm.id && r.manager)
```
[the remark this answers is not in the document]
Comment (gbdubs) [reply frontend/pages/initiative/[id]/index.vue:37]: Great catch, thanks!

File: frontend/pages/initiative/[id]/relationships.vue (hunk 1 above)
Comment (gbdubs) [reply frontend/pages/initiative/[id]/relationships.vue:32]: Line 44! It's just never used to add someone, but it can be used to remove them :)

Comment (bcspragu) [reply frontend/pages/initiative/[id]/relationships.vue:32]: Lol no no no, you never use `manager` _in this function_, so line 44 ain't doing nothing

**Review by bcspragu — APPROVED** [review]

**Pull request merged**